### PR TITLE
feat(doc): publish documentation guidelines

### PR DESCRIPTION
**Pull request by RoRoJ**
This PR makes public the Scaleway Documentation guidelines.

### CODE REVIEW

File: guidelines.mdx [hunk 1]
```diff
@@ -0,0 +1,517 @@
+---
+meta:
+  title: Scaleway Documentation Guidelines
+  description: Discover comprehensive guidelines on writing and formatting for our documentation site. Ensure consistency and clarity with our style guide, covering grammar, terminology, and best practices.
+content:
+  h1: Scaleway Documentation Guidelines
+  paragraph: Discover comprehensive guidelines on writing and formatting for our documentation site. Ensure consistency and clarity with our style guide, covering grammar, terminology, and best practices.
+tags: style-guide guidelines wording grammar
```
Comment (RoRoJ) [inline guidelines.mdx:8]: Should we add validation/posted date?

File: guidelines.mdx [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+
+We do not accept contributions to Scaleway Documentation in languages other than English.
+
+### Tone
+
+The tone of Scaleway Documentation is generally **formal**. Do not use the **passive voice**, **"please"** or **"thank you"**, **the future tense** or **contractions**.
+
+| ✅ DO WRITE                                | ❌ DON'T WRITE                                  |
+| ------------------------------------------ | ----------------------------------------------- |
+| Click **DELETE**  to delete the Instance.  | The Instance is deleted by clicking **DELETE**. |
```
Comment (bene2k1) [inline guidelines.mdx:37]:
```suggestion
| Click **DELETE** to delete the Instance.  | The Instance is deleted by clicking **DELETE**. |
```

File: guidelines.mdx [hunk 3]
```diff
@@ -0,0 +1,518 @@
...
+| API/CLI          | Presents information and tips for creating and configuring a Scaleway product via both official and third-party APIs, CLIs, SDKs and other developer tools, e.g. Terraform, e.g. [Using Object Storage with the AWS-CLI](/storage/object/api-cli/object-storage-aws-cli/). These pages complement the [main developer reference documentation](https://www.scaleway.com/en/developers/).                           |
+| Additional Content | Provides detailed information on specific subjects and features associated with a Scaleway product, e.g. [Optimize your Object Storage performance](/storage/object/reference-content/optimize-object-storage-performance/)                                            |
+| Tutorial           | Explains how to use a Scaleway product with a third-party tool, or how to use multiple Scaleway products together for a specific use-case, e.g. [Setting up a Proxmox cluster on Elastic Metal servers using Private Networks](/tutorials/setup-elastic-metal-proxmox-cluster-with-private-networks/)                                            |
```
Comment (bene2k1) [inline guidelines.mdx:65]: Maybe put another example here, this tuto is very complex and maybe not the best example

**Review by bene2k1 — APPROVED** [review]

File: guidelines.mdx [hunk 4]
```diff
@@ -0,0 +1,517 @@
...
+### Tone
+
+The tone of Scaleway Documentation is generally **formal**. Do not use the **passive voice**, **"please"** or **"thank you"**, **the future tense** or **contractions**.
+
+| ✅ DO WRITE                                | ❌ DON'T WRITE                                  |
+| ------------------------------------------ | ----------------------------------------------- |
+| Click **DELETE**  to delete the Instance.  | The Instance is deleted by clicking **DELETE**. |
+| Tick the box to accept the conditions.     | Please tick the box to accept the conditions.   |
+| Select **Metrics**. The Grafana dashboard opens. | Select **Metrics**. The Grafana dashboard will open. |
+| You cannot leave the field empty.          | You can't leave the field empty.                |
```
Comment (bene2k1) [inline guidelines.mdx:40]: With this example, shouldn't be "DON'T WRITE" -> "DO NOT WRITE" ?

File: guidelines.mdx (hunk 1 above)
Comment (bene2k1) [reply guidelines.mdx:8]: Validation date might be useful to showcase the document itself isn't outdated.
Posted may not be necessary, I think.

File: guidelines.mdx (hunk 3 above)
Comment (bene2k1) [reply guidelines.mdx:65]:
```suggestion
| Tutorial           | Explains how to use a Scaleway product with a third-party tool, or how to use multiple Scaleway products together for a specific use-case, e.g. [Creating WordPress Instances with the Scaleway CLI
](/tutorials/create-wordpress-instances-cli/)                                            |
```

File: guidelines.mdx [hunk 5]
```diff
@@ -501,7 +503,7 @@ Start by using the macro `<Macro id="requirements" />` or type the macro text ma
 
 Continue by listing the necessary products, resources or items users must have. Remember to start each item in a way that it forms a coherent sentence when combined with the intro macro (`To complete the actions presented below, you must have: ...`).
 
-| DO WRITE ✅ | DON'T WRITE ❌ |
+| DO WRITE ✅ | DO NOT WRITE ❌|
```
Comment (bene2k1) [inline guidelines.mdx:506]:
```suggestion
| ✅ DO WRITE | ❌ DO NOT WRITE |
```

File: guidelines.mdx [hunk 6]
```diff
@@ -0,0 +1,519 @@
...
+1. Click **Instances** in the **Compute** section of the Scaleway console. The **Instances dashboard** displays.
+2. Click the Instance whose flexible IP address you want to detach. The **Overview page** for that Instance displays.
+3. Click **Detach** <Icon name="unlink" /> next to the flexible IP address you want to detach.
+
+❌ **BAD**
+
+1. Start by clicking **Instances** in the **Compute** section of the Scaleway console.
+2. You will see the **Instances dashboard**.
+3. Next, you need to find the Instance whose flexible IP address you want to detach, and click on it. You'll see the **Overview page** for the Instance. To finish the process, click **Detach** <Icon name="unlink" /> next to the flexible IP address you want to detach.
+
```
Comment (nerda-codes) [inline guidelines.mdx:122]: Maybe we can put the tip under the "numbered steps" part (after the list), as I first thought the box was part of the "Bad" section

File: guidelines.mdx [hunk 7]
```diff
@@ -0,0 +1,519 @@
...
+| Syntax                                                              | Renders as                                                |
+| ------------------------------------------------------------------- | ---------------------------------------------------------- |
+| ````Use the toggle <Icon name="toggle" /> to activate the feature````  | Use the toggle <Icon name="toggle" /> to activate the feature |
+ 
+See the [full list of available icons](/#inline-console-icons).
+
+### Messages
+
+Three types of message box are available, to use as follows:
```
Comment (nerda-codes) [inline guidelines.mdx:242]:
```suggestion
Three types of message boxes are available, to use as follows:
```

File: guidelines.mdx [hunk 8]
```diff
@@ -0,0 +1,519 @@
...
+| Organization drop-down menu | <Lightbox src="scaleway-org-dropdown.webp" alt=""  />          |
+| $PRODUCT_NAME dashboard, e.g. Object Storage dashboard | <Lightbox src="scaleway-console-dashboard.webp" alt=""  />  |
+| $PRODUCT_NAME creation page ( NOT `splashscreen`), e.g. Instances creation page | <Lightbox src="scaleway-instances-creation-page.webp" alt=""  /> | 
+| $PRODUCT_NAME creation wizard, e.g. Load Balancer creation wizard | | <Lightbox src="scaleway-creation-wizard.webp" alt=""  /> |           
+
+### Product and feature names
+
+When referring to Scaleway products and features, follow the wording and capitalization that is used on the Scaleway console. See also the [capitalization rules](#capitalization).
+
+A wording handbook is available internally, with a full list of correct product and feature naming. Contact a member of the Scaleway Documentation Team for more details.
```
Comment (RoRoJ) [inline guidelines.mdx:409]:
```suggestion
A wording handbook is available internally, with a full list of correct product and feature naming. Contact a member of the Scaleway Documentation team for more details.
```

**Review by nerda-codes — APPROVED** [review]

File: guidelines.mdx [hunk 9]
```diff
@@ -0,0 +1,520 @@
...
+| ------------------------------------------------------------------- | ---------------------------------------------------------- |
+| ````Open the [Scaleway console](https://console.scaleway.com/)````  | Open the [Scaleway console](https://console.scaleway.com/) |
+
+- Include links to **other Scaleway Documentation pages** to:
+  - Avoid repeating instructions that already exist on an existing page
+  - Help the user find other useful and related content
+  - Link to concept definitions that may not be immediately comprehensible
+- Include links to **external pages** to:
+  - Help the user find third-party tools and resources that are mentioned in documentation
+  - Provide information that is not already contained within Scaleway Documentation. Favour sites such as Wikipedia, which are stable, neutral and open-source.
```
Comment (RoRoJ) [inline guidelines.mdx:193]:
```suggestion
  - Provide information that is not already contained within Scaleway Documentation. Favor sites such as Wikipedia, which are stable, neutral and open-source.
```

File: guidelines.mdx [hunk 10]
```diff
@@ -0,0 +1,520 @@
...
+Do not forget to replace `<YOUR INSTANCE NAME>` with the name of your Instance.  
+```` 
+
+Renders as:
+
+Do not forget to replace `<YOUR INSTANCE NAME>` with the name of your Instance.
+
+### Links
+
+Use native markdown syntax for links:
```
Comment (RoRoJ) [inline guidelines.mdx:181]:
```suggestion
Use native Markdown syntax for links:
```

File: guidelines.mdx [hunk 11]
```diff
@@ -0,0 +1,520 @@
...
+
+### Inline console icons
+
+Use a console icon when referring in the documentation to the equivalent icon in the Scaleway console.
+
+| Syntax                                                              | Renders as                                                |
+| ------------------------------------------------------------------- | ---------------------------------------------------------- |
+| ````Use the toggle <Icon name="toggle" /> to activate the feature````  | Use the toggle <Icon name="toggle" /> to activate the feature |
+ 
+See the [full list of available icons](/#inline-console-icons).
```
Comment (RoRoJ) [inline guidelines.mdx:239]:
```suggestion
See the [full list of available icons](/components/docs/#inline-console-icons).
```

File: guidelines.mdx [hunk 12]
```diff
@@ -0,0 +1,520 @@
...
+
+When referring to different aspects of the Scaleway console, use the following terms:
+
+| Console element             | Screenshot                                                     |
+|-----------------------------|----------------------------------------------------------------|
+| Side menu                   | <Lightbox src="scaleway-console-side-menu.webp" alt=""  />     |
+| Organization drop-down menu | <Lightbox src="scaleway-org-dropdown.webp" alt=""  />          |
+| $PRODUCT_NAME dashboard, e.g. Object Storage dashboard | <Lightbox src="scaleway-console-dashboard.webp" alt=""  />  |
+| $PRODUCT_NAME creation page ( NOT `splashscreen`), e.g. Instances creation page | <Lightbox src="scaleway-instances-creation-page.webp" alt=""  /> | 
+| $PRODUCT_NAME creation wizard, e.g. Load Balancer creation wizard | | <Lightbox src="scaleway-creation-wizard.webp" alt=""  /> |
```
Comment (ldecarvalho-doc) [inline guidelines.mdx:404]: is there supposed to be an extra one here?

**Review by ldecarvalho-doc — APPROVED** [review]

File: guidelines.mdx [hunk 13]
```diff
@@ -0,0 +1,520 @@
...
+
+### Page structure
+
+Scaleway Documentation pages should conform to the following structure:
+
+| PAGE TYPE          | DESCRIPTION                                                                              |
+| -------------------| ---------------------------------------------------------------------------------------- |
+| [Metadata](#metadata)           | At the top of the page, used for indexing, search and the generation of the page title (H1). The `dates` section is crucial for tracking and displaying when pages were first created and last updated. |
+| [Introduction](#introduction)       | A few sentences (or short paragraphs), that briefly explain the purpose of the documentation and give any necessary preliminary information. This will be displayed in search results for the page in Scaleway's internal search engine. |
+| [Requirements](#requirements)       | Bullet points stating what the user must already have accomplished before starting any instructions on this documentation page. Necessary for How tos, Quickstarts and Tutorials. May not be relevant for other page types, e.g. FAQ, Concepts. |
```
[the remark this answers is not in the document]
Comment (ldecarvalho-doc) [reply guidelines.mdx:86]: nevermind, just saw that you did at the end, maybe link to that anchor?

File: guidelines.mdx [hunk 14]
```diff
@@ -0,0 +1,520 @@
...
+
+If you are creating a new page, it may be easiest to duplicate an existing page and adapt it to your requirements. See the raw files on our GitHub repo for this purpose, for example [this tutorial](https://raw.githubusercontent.com/scaleway/docs-content/main/tutorials/install-openvpn/index.mdx).
+
+More information about the different sections of each page can be found by following the links in the table above.
+
+## Page content 
+
+### Headers
+
+- There can be only one H1 on each page (the page title, defined in the [metadata](#metadata-example-and-guidance))
```
Comment (RoRoJ) [inline guidelines.mdx:97]:
```suggestion
- There can be only one H1 on each page (the page title, defined in the [metadata](#metadata))
```

File: guidelines.mdx (hunk 12 above)
Comment (RoRoJ) [reply guidelines.mdx:404]: Well spotted! Thanks

**Pull request merged**